calico/openstack/t_etcd.py

```diff
@@ -352,3 +358,5 @@ def provide_felix_config(self):
         # different things.  Here we provide the prefix that Neutron
         # uses.
         self.client.write('/calico/config/InterfacePrefix', 'tap')
+        # TODO Set this flag only once we're really ready!
+        self.client.write("/calico/config/Ready", 'true')
```

Comment:
@neiljerram We need to code to write this flag after the etcd database is properly in-sync but I wasn't sure where to put it int he plugin code so I wrote it by default.